codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodec.java

```diff
@@ -191,11 +193,39 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
         }
     }
 
+    private void onStreamIdResponse(Http2OutboundStreamIdResponse resp) {
+        assert isOutgoing(resp.streamId());
+
+        if (!(resp.cookie() instanceof Http2StreamChannel)) {
+            throw new IllegalArgumentException("cookie must be of type child channel.");
+        }
+        Http2StreamChannel childChannel = (Http2StreamChannel) resp.cookie();
+        childChannel.streamId(resp.streamId());
+        Http2StreamChannel existing = childChannels.put(resp.streamId(), childChannel);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "existing". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1854)